### PR TITLE
Add some logs

### PR DESCRIPTION
**Pull request by itdove**
Other logs will come along.
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: pkg/cmd/accept/exec.go [hunk 1]
```diff
@@ -70,10 +72,14 @@ func (o *Options) run() error {
 func (o *Options) runWithClient(kubeClient *kubernetes.Clientset, clusterClient *clusterclientset.Clientset) (err error) {
 	for _, clusterName := range o.values.clusters {
 		if o.wait == 0 {
-			_, err = o.accept(kubeClient, clusterClient, clusterName)
+			var csrApproved bool
+			csrApproved, err = o.accept(kubeClient, clusterClient, clusterName, false)
+			if err == nil && !csrApproved {
+				err = fmt.Errorf("no CSR to approve for cluster %s", clusterName)
```
Comment (qiujian16) [inline pkg/cmd/accept/exec.go:78]: we did not check case with err != nil here...And the output message is here with line 150. You may want to add more details?

Comment (itdove) [reply pkg/cmd/accept/exec.go:78]: yes we check err != nil at like 85.
The line 150 is displayed only when waitMode...  so it display continuesly until either the wait time is expired or the accept succeed.

Comment (qiujian16) [reply pkg/cmd/accept/exec.go:78]: I see, thanks

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**